README.md

```diff
@@ -1,63 +1,86 @@
 # Agendash
-[![Build Status](https://travis-ci.org/agenda/agendash.svg)](https://travis-ci.org/agenda/agendash)
-<a href="https://slackin-ekwifvcwbr.now.sh/"><img src="https://slackin-ekwifvcwbr.now.sh/badge.svg" alt="Slack Status"></a>
-[![Known Vulnerabilities](https://snyk.io/test/github/agenda/agendash/badge.svg?targetFile=package.json)](https://snyk.io/test/github/agenda/agendash?targetFile=package.json)
 
-A Dashboard for [Agenda](https://github.com/agenda/agenda)
+A Dashboard for [Agenda](https://github.com/agenda/agenda).
```

Comment:
Changed to maybe more boring one from agendash-v2 original headline (https://github.com/agenda/agendash-v2/blob/master/README.md) but figured any terms like that might not age well over the years. 😅  Happy to put it back.